pyHalo/Halos/HaloModels/ULDM.py

```diff
@@ -159,14 +161,17 @@ def _rescaled_tnfw_params(self, tnfw_params, uldm_params):
         factor = (M_nfw - M_uldm) / M_nfw
 
         if factor < 0:
-            print(factor)
             raise ValueError('The resulting composite NFW profile mass is negative, tweak your parameters, factor = ' + str(factor)
                             + ', M_uldm = ' + str(M_uldm) + ', M_nfw = ' + str(M_nfw))
         else:
             pass
 
         tnfw_params['alpha_Rs'] *= factor
         return tnfw_params
+    
+    def _zeta(self,z):
+        Om_z = Cosmology().astropy.Om(z)
```

Comment:
Can you use the instance of cosmology that exists in the instance of LensCosmo (_lens_cosmo_instance) just to make sure the same cosmological parameters are being used throughout? just self._lens_cosmo_instance.cosmo

Reply:
Fixed